apps/services/license-api/src/app/modules/license/clients/firearmLicense/firearmLicenseMapper.ts

```diff
@@ -0,0 +1,88 @@
+import {
+  FirearmProperty,
+  FirearmPropertyList,
+  LicenseInfo,
+} from '@island.is/clients/firearm-license'
+import format from 'date-fns/format'
+import { format as formatNationalId } from 'kennitala'
+
+const formatDateString = (dateTime: string) =>
+  dateTime ? format(new Date(dateTime), 'dd.MM.yyyy') : ''
+
+const parsePropertyForPkpassInput = (properties?: Array<FirearmProperty>) => {
+  if (!properties?.length) return 'Engin skráð skotvopn'
+
+  const propertyString = properties
+    .map(
+      (property) =>
+        `▶︎ ${property.typeOfFirearm} ${property.name}.\r\nHlaupvídd ${property.caliber}.\r\nNúmer ${property.serialNumber}.`,
+    )
+    .join('\r\n\r\n')
+
+  return propertyString
+}
```

Comment:
Does this belong in the backend?
Also, `\r\n` :-1:   `\n` :+1:

Reply:
It'll be moved to a new client in another [PR](https://github.com/island-is/island.is/pull/9718/)